src/entities/inline-fragments/specs/InlineTree.integration.spec.ts

```diff
@@ -0,0 +1,373 @@
+import { RootInlineNode } from '../RootInlineNode';
+import { TextNode } from '../TextNode';
+import { createInlineToolData, createInlineToolName } from '../FormattingNode';
+
+describe('Inline fragments tree integration', () => {
```

Comment:
```suggestion
describe('Inline fragments tree integration', () => {
```

Reply:
It seems you forgot your suggestion here